contracts/contracts/coordination/IAccessControler.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.8.0;
+
+interface IAccessController {
```

Comment:
😆 ✔️   `IAccessController` -> `IRitualAuthorizer`

Reply:
Is it RitualAuthorizer? The functionality here is to ascertain if the actor creating conditions/access control policies for a given ciphertext is authorized or not. This is related to the Enrico naming discussion, actually, because we depend on having a proper name for that. Maybe something like `PolicyAuthorController`? I don't know, this requires a wider discussion with everyone.